charts/nextcloud/templates/deployment.yaml

```
@@ -409,7 +409,7 @@ spec:
       {{- else if .Values.postgresql.enabled }}
       initContainers:
       - name: postgresql-isready
-        image: {{ .Values.postgresql.image.repository }}:{{ .Values.postgresql.image.tag }}
+        image: {{ .Values.postgresql.image.registry }}/{{ .Values.postgresql.image.repository }}:{{ .Values.postgresql.image.tag }}
```

why not wrote `repository: docker.io/image` instead of 

```yaml
registry: docker.io
repository: image
```

The proposed change would't work for the case where you dont want to provide a registry. It would actually be a breaking change. 
In that case it would make sense to provide a default:
```suggestion
        image: {{ default "docker.io" .Values.postgresql.image.registry }}/{{ .Values.postgresql.image.repository }}:{{ .Values.postgresql.image.tag }}
```